src/Core/Framework/Resources/views/documents/base.html.twig

```diff
@@ -129,7 +129,7 @@
                             {{ config.companyAddress }}<br><br>
                         {% endblock %}
                     </span>
-                    {# todo support companys #}
+                    {{ billingAddress.company }}<br>
```

Comment:
what if there is no company? then there would be empty whitespace, wouldn't it?

Reply:
you are right, fixed it